_posts/2018-09-26-incident-report-es.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Reporte de incidente de Bitcoin ABC (17 de Septiembre de 2018)
+multiLangId: 2018-09-26-incident-report
+lang: es
+---
+
+Este documento contiene información en relación a nuestra respuesta a una vulnerabilidad crítica que afecta a todas las versiones de Bitcoin ABC, versiones de Bitcoin Core 0.14.3 a 0.16.2 y varias versiones derivadas de la versión base de Bitcoin Core como Litecoin, Dash y otras. Medidas apropiadas han sido tomadas para mitigar el impacto de esta vulnerabilidad. El presente documento es proporcionado solamente con fines informativos.
+
+Bitcoin ABC [0.18.2](https://download.bitcoinabc.org/0.18.2/) corrige esta vulnerabilidad y se insta a todos los operadores de nodos a actualizar tan pronto como sea posible.
+
+El 17 de Septiembre de 2018, desarrolladores de Bitcoin ABC fueron notificados de forma anónima sobre la vulnerabilidad. Si explotada, la vulnerabilidad podría haber sido utilizada como un vector DDoS (mediante un crash de nodo) y/o un método para inflar el suministro de monedas. Al momento de la divulgación, Bitcoin ABC solo se encontraba en conocimiento del vector DDoS. El vector de inflación de monedas fue descubierto poco después de la divulgación y corrección del error en Bitcoin ABC.
+
+## Descripción
+
+Al momento de la divulgación, se descubrió una vulnerabilidad en Bitcoin Core y Bitcoin ABC que permitiría a un minero crear un bloque con una transacción conteniendo el gasto de una misma entrada varias veces. Esto provocaría una aserción en el código de validación de bloque, causando un crash del nodo. Dicho bloque especialmente diseñado podría ser enviado a cualquier nodo de la red con la intención de atacarlo y desactivarlo, permitiendo el comportamiento de denegación-de-servicio (*denial-of-service*).
+
+La vulnerabilidad fue inicialmente introducida en Bitcoin Core 0.14, cuyos detalles se proporcionan [aquí](https://bitcoincore.org/en/2018/09/20/notice/). Dado que Bitcoin ABC comenzó como un *software fork* de Bitcoin Core después de la introducción de la vulnerabilidad, ABC también la contenía.
+
+Más información sobre el error descubierto se detalla [aquí](https://medium.com/@awemany/600-microseconds-b70f87b0b2a6) por la persona que lo reveló.
+
+Después de la vulnerabilidad ser corregida, se descubrió que el mismo error podría explotarse para inflar el suministro de monedas si un minero incluyera una transacción sin-difusión (*non-broadcast*) con entradas duplicadas en el bloque.
+
+## Impacto
+
+La vulnerabilidad descrita anteriormente no fue explotada en la red principal (*mainnet*). Esto se puede verificar examinando todas las transacciones en blockchain y viendo que ninguna transacción contiene entradas duplicadas.
```

Comment:
**explotada** no se de donde viene bien esa traducción. No se si se quiso poner **expuesta** o **puesta** (poner).

Reply:
The English version is here if that helps: https://www.bitcoinabc.org/2018-09-26-incident-report/
The intent is to say that the vulnerability was not used by anyone, so there was no damage done.

Reply:
"explotada" is correct. Check with the rest of my co-workers and the term is correct.

Reply:
thanks!